lib/turkish_cities/city.rb

```diff
@@ -9,6 +9,10 @@ class City
   CITY_LIST = YAML.load_file('lib/turkish_cities/data/cities.yaml')
 
   def find_by_id(plate_number)
+    unless plate_number.to_i.between?(1, 81)
```

Comment:
`plate_number` burada `nil` gelme ihtimali var mi? varsa onu da kontrol etmek gerekecek.

Reply:
nil.to_i = 0 döndüğü için aslında kontrol etmiş oluyoruz